src/app/devices/my-devices-details/my-devices-details.component.html

```diff
@@ -0,0 +1,4 @@
+<p>
+  my-devices-details works!
+</p>
+<app-qrcode-reader (callback)="showQrCode($event)"></app-qrcode-reader>
```

Comment:
tutaj powinien być inny komponent nie `qrcode-reader` ale komponent stworzony przez Joanne :) , ale to można już będzie zmienić w kolejnym kroku